arbeitszeit_web/api_presenters/response_errors.py

```diff
@@ -0,0 +1,28 @@
+from abc import ABC, abstractproperty
+
+from arbeitszeit_web.api_presenters.interfaces import JsonDict, JsonString, JsonValue
+
+
+class ApiResponseError(ABC):
```

Comment:
Do you think that this could be a `Protocol` instead? This would allow us to implement the `ApiResponseError` without inheriting from this `ABC`.